Orange/widgets/data/owcsvimport.py

```diff
@@ -627,6 +669,17 @@ class Error(widget.OWWidget.Error):
             "might be binary"
         )
 
+    class Warning(widget.OWWidget.Warning):
+        performance_warning = widget.Msg(
+            "Categorical variables with >100 values may decrease performance.")
+        renamed_vars = widget.Msg("Some variables have been renamed "
+                                  "to avoid duplicates.\n{}")
+        numeric_cast = widget.Msg('Loading data as numeric and changing it to text'
+                                  'may result in altered values.\n'
+                                  'For example, 001 turns into 1.\n'
+                                  'Change the variable type to String in '
+                                  'the Import Options to avoid this.')
```

Comment:
This warning is too long (but I'll admit, I have no better alternative either). In the widget it looks like it's just one line and it leaves the user confused. I didn't see there's more text until I checked it here. Also, when hovering over the warning, only the missing part of the warning showed in the tooltip? That is weird - is it intentional? 🤔 If we have long warnings, then there must be eliding (...) so that the user knows to look for more information, IMO.

Also, newline character is missing at the end of the first line (or a space, your choice).

Reply:
This seems like a general problem, maybe we should put a little down arrow at the end of warnings with more details